### PR TITLE
WIP: Disambiguation interface

### PR DESCRIPTION
**Pull request by atsikiridis**
This still needs:
- some cleaning (remove redundant code) requirements have changed some times
- grouping per taskid in aidRESULTS, so that we keep all tortoise results and not just the last
- what happens when the curator thinks everything is ok?

Even when these are adressed, as @glouppe and @kaplun  have mentioned this should be deployed on test first.

### CODE REVIEW

File: modules/miscutil/lib/upgrades/invenio_2014_07_31_new_aidDISAMBIGUATIONLOG.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from invenio.dbquery import run_sql
+
+
+def info():
```
Comment (kaplun) [inline modules/miscutil/lib/upgrades/invenio_2014_07_31_new_aidDISAMBIGUATIONLOG.py:4]: @atsikiridis please add corresponding table definitions and operations into `tabcreate.sql`, and `tabdrop.sql`.

File: modules/miscutil/lib/upgrades/invenio_2014_07_31_new_aidDISAMBIGUATIONLOG.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                `taskid` BIGINT( 16 ) UNSIGNED NOT NULL ,
+                `surname` VARCHAR( 255 ),
+                `phase` VARCHAR( 255 ),
+                `progress` FLOAT( 8 ),
+                `args` longblob NOT NULL,
+                `start_time` datetime,
+                `end_time` datetime,
+                `status` enum('SCHEDULED', 'RUNNING', 'SUCCEEDED', 'FAILED',
+                'KILLED', 'MERGED') NOT NULL
+                ) ENGINE=MyISAM""")
```
Comment (kaplun) [inline modules/miscutil/lib/upgrades/invenio_2014_07_31_new_aidDISAMBIGUATIONLOG.py:21]: Indexes are missing here. Shouldn't at least be some `primary key` or in general `index` over  `taskid`?

**Comment by atsikiridis** [comment]
@kaplun thank you for taking a look. I will try to address your comments ASAP.

**Comment by kaplun** [comment]
I just noticed this is based on a giiiiiiiiaaaaaaaiant branch that is not based on prod. Can it be rebased on prod for proper review?

**Pull request closed without merging**